vshard/replicaset.lua

```diff
@@ -1033,6 +1033,9 @@ local replica_mt = {
         safe_uri = function(replica)
             local uri = luri.parse(replica.uri)
             uri.password = nil
+            if uri.ipv6 then
```

Comment:
Please, fix the commit message:

1. Commit title must have a prefix, which should state the affected subsystem. I.e. in your case you should have `replicaset: support ipv6 uri`.
2. Add meaningful commit message: describe, what was done and why.
3. At the end of the commit message, please, state, why doc and test was skipped: `NO_DOC=...`, `NO_TEST=...`.

For example please check some commits, which are already in master.

Reply:
Done